api/PerformanceEntry.json

```diff
@@ -284,10 +296,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/PerformanceEntry/toJSON",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "45"
```

Comment:
This is suspect because of https://github.com/mdn/browser-compat-data/issues/7844. The collector doesn't use commit mapping at all of course, but I double checked to confirm 45 anyway:
- `PerformanceEntry.prototype.hasOwnProperty('toJSON')` tested in Chrome 44 and 45
- https://chromium.googlesource.com/chromium/src/+/64d41ea9eeb27ea0897a190b62e44e2f1d57ed48 is Jul 9th, 2015
- https://www.chromium.org/developers/calendar/ branch point for 45 is Jul 10th, 2015